include/picongpu/plugins/xrayScattering/beam/beamShapes/shapes.hpp

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2020 Pawel Ordyna
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+# pragma once
+
+#include "picongpu/plugins/xrayScattering/beam/beamShapes/ConstShape.hpp"
```

Comment:
Yes, so the plan was to have more. I reduced it to just `ConstShape.hpp` since with the coordinate transform still not fixed, anything else can't be really used.

Reply:
Fine, just leave it as is.